Canvas/GradesWidget/Info.plist

```diff
@@ -20,6 +20,11 @@
 	<string>6.3.6</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```

Comment:
was this only necessary for testing against local canvas?

Reply:
Yes